Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarkDetailsView.swift

```diff
@@ -20,10 +21,12 @@ class BookmarkDetailsView: AddEditHeaderView, BookmarkFormFieldsProtocol {
     
     let urlTextField: UITextField? = UITextField().then {
         $0.placeholder = Strings.BookmarkUrlPlaceholderText
-        $0.keyboardType = .URL
+        //$0.keyboardType = .URL
```

Comment:
I don't think we should remove it, most people is going to add regular bookmarks, not bookmarklets.
Is this keyboard type causing problems with adding bookmarklets?

Reply:
> I don't think we should remove it, most people is going to add regular bookmarks, not bookmarklets.
> Is this keyboard type causing problems with adding bookmarklets?

Yeah. It doesn't let you type in a bookmarklet manually because the URL type doesn't support `"`, `{`, `}`, etc.. :(

Reply:
I tested keyboard types in other browsers and:
Safari and Firefox use url keyboard type while Chrome does not, I will talk to product team about it

Reply:
@srirambv